compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -2078,17 +2092,28 @@ generateVRSdInstruction(
                       TR::Node                * n          ,
                       TR::Register            * targetReg  ,   /* VRF */
                       TR::Register            * sourceReg3 ,   /* GPR R3 */
-                      TR::MemoryReference     * mr        )
+                      TR::MemoryReference     * mr         ,
+                      TR::Instruction         * preced)
    {
-   return new (INSN_HEAP) TR::S390VRSdInstruction(cg, op, n, targetReg, sourceReg3, mr);
+   if (mr != NULL) preced = mr->enforceSSFormatLimits(n, cg, preced);
```

Comment:
For all of the changes in S390GenerateInstructions.cpp the memory reference argument cannot be NULL. Can we remove these checks?

Reply:
Whoops did not mean to approve. That was an accident. We'll need to make this change before we can merge as well as the change from Leo above.

Reply:
NULL checks removed.